modules/seqexec/server/src/main/scala/seqexec/server/gcal/GcalKeywordReader.scala

```diff
@@ -41,25 +41,25 @@ object GcalKeywordsReaderEpics {
     def filter: F[String] = sys.filter.safeValOrDefault
 
     def lamp: F[String] = {
-      val noValue = none[String].pure[F]
-      val onCheck: BinaryOnOff => Boolean = _ === BinaryOnOff.ON
-      val ar   = sys.lampAr.delay.map(_.exists(onCheck)).ifM("Ar".some.pure[F], noValue)
-      val cuAr = sys.lampCuAr.delay.map(_.exists(onCheck)).ifM("CuAr".some.pure[F], noValue)
-      val ir   = Nested(sys.lampIr.delay).map{
-        case BinaryOnOff.ON => "IRhigh"
-        case _              => "IRlow"
-      }.value
-      val qh   = sys.lampQH.delay.map(_.exists(onCheck)).ifM("QH".some.pure[F], noValue)
-      val thAr = sys.lampThAr.delay.map(_.exists(onCheck)).ifM("ThAr".some.pure[F], noValue)
-      val xe   = sys.lampXe.delay.map(_.exists(onCheck)).ifM("Xe".some.pure[F], noValue)
+      def onCheck(v: BinaryOnOff): Boolean = v === BinaryOnOff.ON
 
-      ar.orElse(xe).orElse(cuAr).orElse(thAr).orElse(qh).orElse(ir).safeValOrDefault
```

Comment:
This shouldn't work. The intention is that the result must be the first `Some(_)`, or a default value if none. But the `orElse` here is run over `F`, not teh contained `Option`!